.travis.yml

```diff
@@ -21,6 +21,9 @@ env:
     - SPARK_BUILD_URL="https://dist.apache.org/repos/dist/release/spark/spark-${SPARK_VERSION}/spark-${SPARK_VERSION}-bin-hadoop2.7.tgz"
     - SPARK_HOME=$HOME/.cache/spark-versions/$SPARK_BUILD
     - RUN_ONLY_LIGHT_TESTS=True
+    # TODO: This is a temp fix in order to pass tests.
+    # We should update implementation to allow graph construction via C API.
+    - TF_C_API_GRAPH_CONSTRUCTION=0
```

Comment:
For more context, IIRC, i added this as a debug mechanism, i do not think this should make a condition for our testing, according to the commit, https://github.com/tensorflow/tensorflow/commit/0f60a7d10102048625e45c07a59c71c0c4375529 
> One known difference is improved static shape inference, meaning some shape errors will be surfaced during graph construction instead of at runtime.

Which means we might need to update our assumption that the shape will be checked only during run time. As noted in the comments above.

Also, i remember a vague suspicion that we may have ben using different versions of tensorflow in tensorframes and this library. can we confirm if both tensorframes and sparkdl use tf 1.10 ?

Reply:
We didn't release tensorframes with TF 1.10 yet. After the releasing of the new tensorframes we can retest this.